client/client-multi/client-c/src/kaa_event.c

```diff
@@ -495,6 +576,37 @@ static kaa_error_t kaa_event_list_serialize(kaa_event_manager_t *self, kaa_list_
     return KAA_ERR_NONE;
 }
 
+static kaa_error_t kaa_event_listeners_request_serialize(kaa_event_manager_t *self, kaa_platform_message_writer_t *writer, uint16_t *listeners_count)
+{
+    uint16_t count = 0;
+    kaa_list_t *cursor = self->event_listeners_requests;
+    while (cursor) {
+        kaa_event_listeners_request_t *request = (kaa_event_listeners_request_t *) kaa_list_get_data(cursor);
+        if (!request->is_sent) {
+            *((uint16_t *) writer->current) = KAA_HTONS(request->request_id);
+            writer->current += sizeof(uint16_t);
+            *((uint16_t *) writer->current) = KAA_HTONS((uint16_t) request->fqns_count);
+            writer->current += sizeof(uint16_t);
+            int i = 0;
+            for (; i < request->fqns_count; ++i) {
+                size_t fqn_length = strlen(request->fqns[i]);
+                *((uint16_t *) writer->current) = KAA_HTONS((uint16_t) fqn_length);
+                writer->current += sizeof(uint32_t); // fqn length + reserved
+                kaa_error_t error_code = kaa_platform_message_write_aligned(writer, request->fqns[i], fqn_length);
```

Comment:
I suggest using just "error" by default as a variable name instead of "error_code" for consistency around the project.